com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameObjectBuilder.java

```diff
@@ -212,6 +236,36 @@ private PrototypeDesc.Builder transformGo(IResource resource,
                 }
             }
             ComponentDesc.Builder compBuilder = ComponentDesc.newBuilder(cd);
+
+            // #3981
+            // migrate label scale from LabelDesc to ComponentDesc
+            if (isComponentOfType(cd, "label")) {
+                try {
+                    // find the label resource in one of two places:
+                    // * in the build directory if it is generated from an embedded label (see build step above)
+                    // * in the project
+                    IResource labelResource = project.getResource(cd.getComponent());
+                    if (!labelResource.exists()) {
+                        labelResource = labelResource.output();
+                    }
+                    FileReader reader = new FileReader(labelResource.getAbsPath());
+                    LabelDesc.Builder lb = LabelDesc.newBuilder();
+                    if (lb.hasScale()) {
```

Comment:
`lb` was just created, so how can it have scale? Was this code left in an unfinished state?